### ISSUE TITLE
DB Parser can NOT handle multiple DB operation

### ISSUE THREAD

**Issue by sleek-geek** [elbadawyy/Hash-POS#4]
If there are two DB operations in the same the script an error occures.

The error would look like this:

```
Traceback (most recent call last):
  File "items.py", line 44, in <module>
    i.mod_name("1", "bra")
  File "items.py", line 24, in mod_name
    dbHelper.modItem(id, name)
  File "../Helper/dbHelper.py", line 81, in modItem
  File "../Helper/dbHelper.py", line 17, in importDbVars
  File "/usr/lib/python2.7/ConfigParser.py", line 607, in get
    raise NoSectionError(section)
ConfigParser.NoSectionError: No section: 'DataBaseConfig'
```
My guess is that it has something to do with this line:

`config.read('../../Config/DB.conf')`

**Comment by elbadawyy** [comment]
You can now test this issue, I think it has been handled.

**Comment by sleek-geek** [comment]
Issue still exists. I have tried using `os.getcwd()` method last night too, but didn't work and still doesn't.

I have also tried giving the full path to the `config.read('../../Config/DB.conf')`  and turning it to `config.read('/home/user/Code/Hash-POS/Config/DB.conf')` and, not surprisingly, it worked.

I think this issue will be solved when we start working on a proper dictionary for paths and other variables.
